<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSequenceBrowserPlayWidget</class>
 <widget class="QWidget" name="qMRMLSequenceBrowserPlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>82</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>qMRMLSequenceBrowserPlayWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_VcrFirst">
     <property name="toolTip">
      <string>First frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerFirstOrSkipBackward.svg</normaloff>:/Icons/SlicerFirstOrSkipBackward.svg
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrPrevious">
     <property name="toolTip">
      <string>Previous frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerBack.svg</normaloff>:/Icons/SlicerBack.svg
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrPlayPause">
     <property name="toolTip">
      <string>Play/Pause</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerPlay.svg</normaloff>
       <normalon>:/Icons/SlicerPause.svg</normalon>:/Icons/SlicerPlay.svg
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrNext">
     <property name="toolTip">
      <string>Next frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerForward.svg</normaloff>:/Icons/SlicerForward.svg
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrLast">
     <property name="toolTip">
      <string>Last frame</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerLastOrSkipForward.svg</normaloff>:/Icons/SlicerLastOrSkipForward.svg
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_VcrPlaybackRate">
     <property name="suffix">
      <string>fps</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrLoop">
     <property name="toolTip">
      <string>Loop playback</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerLoop.svg</normaloff>:/Icons/SlicerLoop.svg
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_VcrRecord">
     <property name="toolTip">
      <string>Record proxy nodes modifications continuously</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerRecordOrRecording.svg</normaloff>:/Icons/SlicerRecordOrRecording.svg
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Snapshot">
     <property name="toolTip">
      <string>Record snapshot of current state of all proxy nodes</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../qSlicerSequenceBrowserModuleWidgets.qrc">
       <normaloff>:/Icons/SlicerCapture.svg</normaloff>:/Icons/SlicerCapture.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_VcrFirst</tabstop>
  <tabstop>pushButton_VcrPrevious</tabstop>
  <tabstop>pushButton_VcrPlayPause</tabstop>
  <tabstop>pushButton_VcrNext</tabstop>
  <tabstop>pushButton_VcrLast</tabstop>
  <tabstop>doubleSpinBox_VcrPlaybackRate</tabstop>
  <tabstop>pushButton_VcrLoop</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Resources/qSlicerSequenceBrowserModule.qrc"/>
  <include location="../qSlicerSequenceBrowserModuleWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
